blint/utils.py

```diff
@@ -1696,3 +476,28 @@ def cleanup_list_lief_errors(d):
             entry = dl
         new_lst.append(entry)
     return new_lst
+
+
+def create_component_evidence(method_value: str) -> ComponentEvidence:
+    """
+    Creates component evidence based on the provided method value.
+
+    Args:
+        method_value: The value of the method used for analysis.
+
+    Returns:
+        ComponentEvidence: The created component evidence.
+    """
+    return ComponentEvidence(
+        identity=Identity(
+            field=FieldModel.purl,
```

Comment:
Let's accept confidence as an argument since there are cases where this could be higher.

Reply:
@prabhu Good idea! Updated.